proto/grpc/federation/federation.proto

```diff
@@ -579,23 +579,23 @@ message ValidationErrorDetail {
   // rule specifies validation rule in CEL.
   string rule = 1;
   // error_info describes the cause of the error with structured details.
-  repeated google.rpc.ErrorInfo error_info = 3;
+  repeated google.rpc.ErrorInfo error_infos = 3;
```

Comment:
If we follow the current naming convention, I believe the field should be named `error_infos` instead of `error_info`, so I renamed it.